src/map/mob.c

```diff
@@ -5630,8 +5631,8 @@ static bool mob_skill_db_libconfig_sub_skill(struct config_setting_t *it, int n,
 
 	int i32 = MSS_ANY;
 	if (mob->lookup_const(it, "SkillState", &i32) && (i32 < MSS_ANY || i32 > MSS_ANYTARGET)) {
-		ShowWarning("%s: Invalid skill state %d for skill %d (%s) in %s %d, defaulting to MSS_ANY.\n",
-			    __func__, i32, skill_id, skill_name, mob_str, mob_id);
+		ShowWarning("%s: Invalid skill state %d for skill %d (%s) in %s %s %d, defaulting to MSS_ANY.\n",
```

Comment:
Please put the ID into brackets to be consistent.

Reply:
did you mean parenthesis? like that?
`in %s %d (%s)`? which would render something like (`in monster 1001 (SCORPION)`) ?

Reply:
I meant you just should do it like you did in the first two messages you changed:
`%s (%d)` -> `<mob_sprite> (<mob_id>)`